src/apps/somm/positions.test.ts

```diff
@@ -148,13 +148,13 @@ describe('hook', () => {
               data: {
                 cellars: [
                   {
-                    id: '0x392b1e6905bb8449d26af701cdea6ff47bf6e5a8-arb',
+                    id: '0x392b1e6905bb8449d26af701cdea6ff47bf6e5a8-arbitrum',
                     shareValue: '1050000',
                     tvlTotal: 10000000,
                     chain: 'arbitrum',
                   },
                   {
-                    id: '0xcellarAddressThatResultsInReadError-arb',
+                    id: '0xc47bb288178ea40bf520a91826a3dee9e0dbfa4c-arbitrum', // mockReadContract will fail on this address
```

Comment:
supernit: what about instead of using a real chain name, use here a deliberately made-up one, like `unknown`, to emphasize it should fail?

Reply:
good idea, updated!